<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<<<<<<< HEAD
<<<<<<< HEAD
<testsuite skipped="6" hostname="QTLSYS1-182" name="tests.QCStore" tests="8" failures="0" timestamp="26 Mar 2019 13:00:03 GMT" time="1577.001" errors="0">
  <testcase name="JQC_AgeStore_Deposit_Return_Refinance_Void_Txn" time="841.321" classname="tests.QCStore"/>
  <testcase name="JQC_AgeStore_Refinance_Rescind_Txn" time="735.680" classname="tests.QCStore"/>
=======
<testsuite skipped="6" hostname="QTLSYS2-48" name="tests.QCStore" tests="7" failures="0" timestamp="02 Apr 2019 05:27:44 GMT" time="806.719" errors="0">
  <testcase name="RefinanceStepup" time="806.719" classname="tests.QCStore"/>
>>>>>>> anoop1.1
=======
<testsuite skipped="6" hostname="QTLSYS2-238" name="tests.QCStore" tests="28" failures="3" timestamp="01 Apr 2019 19:09:40 GMT" time="22403.674" errors="0">
  <testcase name="QC_NewLoan_Promotion_Txn" time="377.401" classname="tests.QCStore"/>
  <testcase name="QC_RefinanceStepDown_Txn" time="1133.943" classname="tests.QCStore"/>
  <testcase name="QC_RefinanceStepDown_Void_Txn" time="1388.917" classname="tests.QCStore"/>
  <testcase name="QC_OpenLoan_NewLoan_Txn" time="444.370" classname="tests.QCStore"/>
  <testcase name="QC_NewLoan_Void_Txn" time="326.327" classname="tests.QCStore"/>
  <testcase name="QC_AgeStore_Void_Txn" time="506.256" classname="tests.QCStore"/>
  <testcase name="QC_Deposit_Menu_Txn" time="740.289" classname="tests.QCStore"/>
  <testcase name="QC_Deposit_GraceDays_Txn" time="834.424" classname="tests.QCStore"/>
  <testcase name="QC_FutureDeposit_DropDown" time="1052.794" classname="tests.QCStore"/>
  <testcase name="QC_Epp_Deposit_Void_Txn" time="1231.775" classname="tests.QCStore"/>
  <testcase name="QC_Epp_PrePayment_Txn" time="1350.343" classname="tests.QCStore"/>
  <testcase name="QC_Epp_Check_Deposit_Clear_Txn" time="1552.275" classname="tests.QCStore"/>
  <testcase name="QC_EppDepositClear_Refund_Txn" time="1510.456" classname="tests.QCStore"/>
  <testcase name="QC_Epp_PrePayment_Return_Txn" time="1351.038" classname="tests.QCStore"/>
  <testcase name="QC_Epp_Return_NSFPayment_Txn" time="1432.047" classname="tests.QCStore"/>
  <testcase name="QC_Epp_Return_ReDeposit_Txn" time="1413.732" classname="tests.QCStore"/>
  <testcase name="QC_Void_FullReturnPayment_Txn" time="1389.679" classname="tests.QCStore"/>
  <testcase name="QC_PartialPayment_RefundVoid_Txn" time="1107.442" classname="tests.QCStore"/>
  <testcase name="QC_Refinance_StepSameURP_Txn" time="851.613" classname="tests.QCStore">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertTrue(Assert.java:44)
at org.testng.Assert.assertTrue(Assert.java:54)
at tests.QCRefinance.refinanceStepSame(QCRefinance.java:730)
at tests.QCStore.QC_Refinance_StepSameURP_Txn(QCStore.java:1457)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- QC_Refinance_StepSameURP_Txn -->
  <testcase name="QC_Refinance_StepSameVoidURP_Txn" time="735.355" classname="tests.QCStore">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertTrue(Assert.java:44)
at org.testng.Assert.assertTrue(Assert.java:54)
at tests.QCRefinance.refinanceStepSame(QCRefinance.java:730)
at tests.QCStore.QC_Refinance_StepSameVoidURP_Txn(QCStore.java:1516)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- QC_Refinance_StepSameVoidURP_Txn -->
  <testcase name="QC_Refinance_StepDownURP_Txn" time="904.619" classname="tests.QCStore">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertTrue(Assert.java:44)
at org.testng.Assert.assertTrue(Assert.java:54)
at tests.QCRefinance.refinanceStepDown(QCRefinance.java:359)
at tests.QCStore.QC_Refinance_StepDownURP_Txn(QCStore.java:1331)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- QC_Refinance_StepDownURP_Txn -->
  <testcase name="QC_Refinance_StepDownVoidURP_Txn" time="768.578" classname="tests.QCStore"/>
>>>>>>> brahmith1.1
  <testcase name="JQC_AgeStore_GraceDays_DCDeposit_Menu_Txn" classname="tests.QCStore">
    <skipped/>
  </testcase> <!-- JQC_AgeStore_GraceDays_DCDeposit_Menu_Txn -->
  <testcase name="JQC_ACH_PartialWORecovery_FullWORecovery_Void_Txn_Testdata" classname="tests.QCStore">
    <skipped/>
  </testcase> <!-- JQC_ACH_PartialWORecovery_FullWORecovery_Void_Txn_Testdata -->
  <testcase name="JQC_AgeStore_GraceDays_DCDeposit_Dropdown_Txn" classname="tests.QCStore">
    <skipped/>
  </testcase> <!-- JQC_AgeStore_GraceDays_DCDeposit_Dropdown_Txn -->
  <testcase name="JQC_AgeStore_DCDeposit_Txn" classname="tests.QCStore">
    <skipped/>
  </testcase> <!-- JQC_AgeStore_DCDeposit_Txn -->
  <testcase name="JQC_NewLoan_Veritec_Txn" classname="tests.QCStore">
    <skipped/>
  </testcase> <!-- JQC_NewLoan_Veritec_Txn -->
  <testcase name="JQC_AgeStore_DCDeposit_Menu_Txn" classname="tests.QCStore">
    <skipped/>
  </testcase> <!-- JQC_AgeStore_DCDeposit_Menu_Txn -->
</testsuite> <!-- tests.QCStore -->
